share/wake/html/timeline_template.html

```diff
@@ -0,0 +1,41 @@
+<!--
+ Copyright 2022 SiFive, Inc.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You should have received a copy of LICENSE.Apache2 along with
+ this software. If not, you may obtain a copy at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ -->
+
+<!doctype html>
+<html>
+<head>
+    <title>Timeline</title>
+    <script type="text/javascript"
+            src="https://unpkg.com/vis-timeline@latest/standalone/umd/vis-timeline-graph2d.min.js"></script>
+    <link href="https://unpkg.com/vis-timeline@latest/styles/vis-timeline-graph2d.min.css" rel="stylesheet"
```

Comment:
You should probably pin these to a specific version and not `latest`, since the next time someone publishes a new version to the internet, it could break previously generated HTML files.

Reply:
Done